Grafana/confd/templates/main-dashboard.json.tpl

```diff
@@ -1303,6 +1303,99 @@
             ],
             "title": "Event Handler Failures",
             "type": "timeseries"
+        },
+        {
+          "id": 47,
+          "gridPos": {
+            "x": 0,
+            "y": 0,
+            "w": 12,
+            "h": 8
+          },
+          "type": "timeseries",
+          "title": "Web response time(response time from nginx auth proxy)",
+          "targets": [
+            {
+              "refId": "A",
+              "editorMode": "code",
+              "expr": "max(aws_applicationelb_target_response_time_p99_00{load_balancer=~\".*-bichard-7-nginx.*\"})",
```

Comment:
Nit: Any chance you can make the graph legend use words rather than aws info? Something like `Avg/max response time` maybe?

![image](https://user-images.githubusercontent.com/6376724/185969069-54342715-c4fd-43d9-bcf2-33cc5c0543ca.png)

Reply:
Yep, resolved in [b2d68f0](https://github.com/ministryofjustice/bichard7-next-infrastructure-docker-images/pull/181/commits/b2d68f068170302ef844c987a6bb339957313f8c)